Bottom: 3213ee2eca2b6c9ee95751032d2412a432383e88
Top:    fc1641c4347c1b027ee862d5eff3749f8c010dcd
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-22 15:56:54 -0500

Refresh of TESTING--VirtualMachine

---
diff --git a/ansible-playbook/group_vars/docker-hosts/repos.yml b/ansible-playbook/group_vars/docker-hosts/repos.yml
index acc5521..8917e7c 100644
--- a/ansible-playbook/group_vars/docker-hosts/repos.yml
+++ b/ansible-playbook/group_vars/docker-hosts/repos.yml
@@ -2,6 +2,7 @@
 
 openssl_image_repo: https://github.com/AndrewCooper/docker-openssl.git
 openldap_image_repo: https://github.com/AndrewCooper/docker-openldap.git
+phpldapadmin_image_repo: https://github.com/AndrewCooper/docker-phpldapadmin.git
 wiki_image_repo: https://github.com/AndrewCooper/docker-mediawiki.git
 mantisbt_image_repo: https://github.com/AndrewCooper/docker-mantisbt.git
 svn_image_repo: https://github.com/AndrewCooper/docker-svn.git
